cli/src/plugins/javadoc/index.ts

```diff
@@ -323,6 +343,32 @@ function makeClassDocPageBody(args: MakeSectionArgs): Node[] {
           .flat(1),
     }),
 
+    ...makeSection({
+      ...args,
+      depth,
+      title: "Enum Constant Detail",
+      shouldMakeSection: () => doc.enumConstants.length !== 0,
+      makeBody: () =>
+        doc.enumConstants
+          .map((doc) => [
+            project.declareEntity({
+              canonicalName: doc.qualifiedName + "_" + doc.name,
```

Comment:
Prefer template literals:
```suggestion
              canonicalName: `${doc.qualifiedName}_${doc.name}`,
```

Reply:
Same as above.

Reply:
Btw, I might prefer to use `.` instead of `_` here. Arbitrary, but that seems more Java-y.